forms/HtmlEditorField.php

```diff
@@ -323,8 +323,12 @@ public function MediaForm() {
 		$fileField->setAttribute('data-multiselect', true);
 		$columns = $fileField->getConfig()->getComponentByType('GridFieldDataColumns');
 		$columns->setDisplayFields(array(
-			'CMSThumbnail' => false,
-			'Name' => _t('File.Name'),
+			'StripThumbnail' => false,
+			'Title' => _t('File.Title'),
+			'Created' => singleton('File')->fieldLabel('Created'),
+		));
+		$columns->setFieldCasting(array(
+			'Created' => 'Date->Nice'
```

Comment:
I'm not exactly sure where the time is being displayed but I'm guessing we are talking about it being displayed in the grid view. As long as the user can drill into the details to access the time I would say that it isn't that necessary (of course there will be edge cases where it would be very useful). A middle ground could be to add the time next to the date but make it lighter in color so it doesn't stand out as much making it secondary information. I think I'd be more inclined to go without it if lightening the time doesn't look good or adds too much clutter.